KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java

```diff
@@ -180,19 +180,63 @@ public void mouseClick(MouseEvent event) {
     }
     
     /**
-     * Roll both of the 6 sided dice and
+     * Roll both of the 6 sided dice and act according to the roll.
+     * 7 Will trigger thief movement, and other values give resources.
+     * The roll is done as 2 1d6 rolls to create the same number rarity as 2 dice give
      */
     private void diceRoll() {
-    
+        int roll; // Holds the dice roll
+        
+        // Roll the first dice
+        roll = (int)(Math.random()*6) + 1;
+        // Roll the second dice and add to the total
+        roll += (int)(Math.random()*6) + 1;
+        
+        System.out.println("Rolled a " + roll);
```

Comment:
Should probably display this on screen somewhere

Reply:
yeah